pkg/ingestor/api/api.go

```diff
@@ -83,6 +91,13 @@ func (g *IngestorAPI) Ingest(_ context.Context, clusterName string, runID string
 		},
 	}
 
+	// We need to flush the cache to prevent warning/error on the overwriting element in cache the  any conflict with the previous ingestion
```

Comment:
I think this comment is missing a few words?

Reply:
yes it does fixing.